README.md

```diff
@@ -35,18 +35,14 @@ This has consequences for reproducibility if you do not store a copy of the data
 
 1.  It's good practice to include the schema and server when you query data. That is, write:
 ```
-select * from permit.vps_owner@sole
+select * from nefsc_garfo.permit_vps_owner@NEFSC_USERS
```

Comment:
are the @ symbols needed?

Reply:
They are not. It's explicit about the location of the database, which is good practice that nobody follows.

Reply:
> Do you note the difference between the description of NEFSCDB1 vs NEFSC_USERS? Potentially making a Server overview md file?

This (NEFSCDB1 vs NEFSC_USERS) depends on your local configuration. On my laptop, ITD set up the connection name "NEFSC_USERS" to point to NEFSCDB1.  On the SAS container, they renamed the "nova" connection name to point to NEFSCDB1.  I will tidy up this text

schema and server 

to schema and connection name.

I think a Server overview file would be a great idea. It's beyond the scope of this PR.